src/xdocs/config_coding.xml

```diff
@@ -1365,24 +1365,26 @@ public class MyClass {
           <tr>
             <td>setterCanReturnItsClass</td>
             <td>
+              Allow to ignore the parameter of a property setter
+              method returning class in which setter is defined.
               Used in conjunction with <code>ignoreSetter</code> property it
               controls rule that recognizes method as a setter. By default
```

Comment:
Would a better description be: `Allow to expand the definition of a setter method to include methods that return the class' instance`
?